_tools/packaging/prepare-deb-build.sh

```diff
@@ -7,6 +7,7 @@ pwd=`dirname $0`
 
 MACKEREL_AGENT_NAME=${MACKEREL_AGENT_NAME:-mackerel-agent}
 MACKEREL_AGENT_VERSION=$(grep -o -e "[0-9]\+.[0-9]\+.[0-9]\+-[0-9]" packaging/deb/debian/changelog | head -1 | sed 's/-.*$//')
+BUILD_DIRECTORY=${BUILD_DIRECTORY:-build/}
```

Comment:
Yes. Fixed.
(I think repeating `/` by trailing `/` is accepted, because it's not affect behavior and is much safer than luck of necessary one.)

(btw your code block is typo for `cp ...` ?)